data_test.go

```diff
@@ -126,7 +126,7 @@ var _ = Describe("data", func() {
 			// write
 			err := testDbMan.updateLSN(testLSN)
 			Expect(err).Should(Succeed())
-			rows, err := testDbMan.getDb().Query(`
+			rows, _ := testDbMan.getDb().Query(`
```

Comment:
Could you keep `err`, and add `Expect(err).Should(Succeed())`?
Thanks

Reply:
Thanks, pushed this update now.